fractal_server/app/models/task.py

```diff
@@ -24,3 +24,10 @@ def parallelization_level(self) -> Optional[str]:
     @property
     def is_parallel(self):
         return bool(self.parallelization_level)
+
+    @property
+    def executor(self) -> Optional[str]:
+        try:
+            return self.default_args["executor"]
+        except KeyError:
+            return None
```

Comment:
i would very much like to avoid having parsl specific stuff in the task definition. For now we can keep it like this, but we should work towards adding objective constraints in the tasks and move the selection of a specific executor to the backends or as close as possible to the backends.

E.g.,
* Task says "I needs 4GB memory"
* backend says "the executor that fits your requirements is `hi-mem-blabla`"

Reply:
OK, let's keep this in mind